toolchain/lc-compile/src/report.c

```diff
@@ -27,13 +27,13 @@ extern int IsDependencyCompile(void);
 ////////////////////////////////////////////////////////////////////////////////
 
 static int s_error_count;
-int s_verbose_level;
+int s_verbose_level = 1;
 int s_is_werror_enabled;
 
 void InitializeReports(void)
 {
     s_error_count = 0;
-    s_verbose_level = 0;
+    s_verbose_level = 1;
```

Comment:
Was changing the default verbosity level a deliberate change? It was deliberately set to 0 by default (and the `-v`/`--verbose` option was added) in order to avoid swamping the default build logs with compiler debugging output.